packages/core/contracts/proxy-scripts/lsp-broker/LspUniswapV2Broker.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
+import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router01.sol";
+
+import "../../common/implementation/FixedPoint.sol";
+
+import "../../financial-templates/long-short-pair/LongShortPair.sol";
+
+/**
+ * @title LspUniswapV2Broker
+ * @notice Helper contract to facilitate batched LSP and UniswapV2 transactions, including Mint+Sell and Mint+LP.
+ */
+contract LspUniswapV2Broker {
+    using FixedPoint for FixedPoint.Unsigned;
```

Comment:
I removed `SafeMath`. We can remove all `SafeMaths` in a future (massive) PR but might as well do it when we have a chance to do it.

Reply:
ye this will save bytecode and gas. save for later in other files